plugins/Transitions/API.php

```diff
@@ -222,7 +222,8 @@ protected function queryFollowingActions($idaction, $actionType, LogAggregator $
         if ($actionType != 'title') {
             // specific setup for page urls
             $types[Action::TYPE_PAGE_URL] = 'followingPages';
-            $dimension = 'if ( idaction_url IS NULL, idaction_name, idaction_url )';
+            $dimension = 'if ( %s.idaction_url IS NULL, %s.idaction_name, %s.idaction_url )';
+            $dimension = str_replace('%s', 'log_link_visit_action', $dimension );
```

Comment:
it's possible to use sprintf but we need to replace `%s` by `%1$s` in all 3 occurrences. Otherwise we get `sprintf(): Too few arguments`. Shall I do it?